lib/stellar_core_commander/process.rb

```diff
@@ -251,12 +251,14 @@ def database_port
 
     Contract None => String
     def dsn
-      base = "postgresql://dbname=#{database_name} "
-      base << " user=#{database_user}" if database_user.present?
-      base << " password=#{database_password}" if database_password.present?
-      base << " host=#{database_host} port=#{database_port}" if database_host.present?
-
-      base
+      if is_sqlite
+        base = database_url
+      else
+        base = "postgresql://dbname=#{database_name} "
+        base << " user=#{database_user}" if database_user.present?
+        base << " password=#{database_password}" if database_password.present?
+        base << " host=#{database_host} port=#{database_port}" if database_host.present?
+      end
```

Comment:
for clarity, add the line `base` at the end of the function (it was there before)

Reply:
Sure, updated.